test/user.test.js

```diff
@@ -150,6 +150,19 @@ describe('User', function() {
       });
     });
 
+    it('Email is required (email case-sensitivity off)', function(done) {
+      User.settings.caseSensitiveEmail = false;
+      User.create({email: undefined, password: '123'}, function(err) {
+        assert(err);
```

Comment:
```js
if (err) return done(err);
```

Since this is newly added code, let's do it the right way please.

When `assert(err)` fails, it reports an assertion error (a stack trace) pointing to this place, hiding any information about the original error that triggered the test failure. With my proposal, the test fails because of the original error, therefore Mocha will print the stack trace pointing to the real source of the problem.